src/App.js

```diff
@@ -908,10 +921,8 @@ export class App extends React.Component<AppProps, AppState> {
         this.focusTrapManager.handleKeyDown(e);
     }
 
-    handleToggleAudioFeedback = (announcementsEnabled: boolean) => {
-        this.setState({
-            announcementsEnabled: announcementsEnabled
-        });
+    handleChangeSoundOptions = (audioEnabled: boolean, announcementsEnabled: boolean, sonificationEnabled: boolean) => {
+        this.setState({ audioEnabled, announcementsEnabled, sonificationEnabled, showSoundOptionsModal: false });
```

Comment:
We aren't super consistent in our ordering of methods in `App`, but could you move the `handleChangeSoundOptions` method up to be right below `handleSoundOptionsModalClose`, so that the sound options modal handlers are grouped together. Thanks.